### PR TITLE
Fix paths in oblivious sort demo

### PR DESCRIPTION
**Pull request by podcastinator**

### CODE REVIEW

File: demo/osort-mem-trace/sort_A.cc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#include "../../enclave/src/common/obl_primitives.h"
+#include "../../include/xgboost/common/obl_primitives.h"
```
Comment (chester-leung) [inline demo/osort-mem-trace/sort_A.cc]: Can we avoid using relative paths here? i.e. changing this to `#include <xgboost/common/obl_primitives.h>`?

File: demo/osort-mem-trace/sort_B.cc [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#include "../../enclave/src/common/obl_primitives.h"
+#include "../../include/xgboost/common/obl_primitives.h"
```
Comment (chester-leung) [inline demo/osort-mem-trace/sort_B.cc]: See above comment

File: demo/osort-mem-trace/sort_A.cc (hunk 1 above)
Comment (podcastinator) [reply demo/osort-mem-trace/sort_A.cc]: Done.

File: demo/osort-mem-trace/sort_B.cc (hunk 2 above)
Comment (podcastinator) [reply demo/osort-mem-trace/sort_B.cc]: Done.

**Review by chester-leung — APPROVED** [review]

**Comment by chester-leung** [comment]
LGTM. Thanks!

**Pull request merged**